src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const SHORT_TERM = 3;
+  const DISCOUNT_FOR_SHORT_TERM = 20;
+  const LONG_TERM = 7;
+  const DISCOUNT_FOR_LONG_TERM = 50;
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days < SHORT_TERM) {
+    return basePrice;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return basePrice - DISCOUNT_FOR_SHORT_TERM;
+  }
+
+  if (days >= LONG_TERM) {
+    return basePrice - DISCOUNT_FOR_LONG_TERM;
+  }
```

Comment:
first condition is redundant
```suggestion
  if (days >= SHORT_TERM && days < LONG_TERM) {
    return basePrice - DISCOUNT_FOR_SHORT_TERM;
  }

  if (days >= LONG_TERM) {
    return basePrice - DISCOUNT_FOR_LONG_TERM;
  }
  
  return basePrice;
```